com.zsmartsystems.zigbee.console/src/main/java/com/zsmartsystems/zigbee/console/ZigBeeConsoleAbstractCommand.java

```diff
@@ -217,6 +262,16 @@ protected String printZclDataType(ZclDataType dataType) {
         return String.format("%-25s", dataType);
     }
 
+    /**
+     * @param cluster a ZCL cluster
+     * @return a String containing information about the cluster, example 'OnOff server cluster 0x00A4'
```

Comment:
Can I suggest changing the format slightly... Instead of ```OnOff server cluster 0x00A4``` -:

```
Reading OnOff server cluster 0x00A4
Supported attributes for OnOff server cluster 0x00A4
Supported generated commands for OnOff server cluster 0x00A4
Attribute XXX was not found in OnOff server cluster 0x00A4
```

How about ```server cluster OnOff (0x00A4)``` -:
```
Reading server cluster OnOff (0x00A4)
Supported attributes for server cluster OnOff (0x00A4)
Supported generated commands for server cluster OnOff (0x00A4)
Attribute XXX was not found in server cluster OnOff (0x00A4)
```

I just think it reads better?

Reply:
I agree, this reads more easily.